noetic/distribution.yaml

```diff
@@ -3997,6 +3997,24 @@ repositories:
       url: https://github.com/ros-planning/moveit.git
       version: noetic-devel
     status: maintained
+  moveit_calibration:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/moveit_calibration.git
+      version: 0.1.0
```

Comment:
It is uncommon (though allowed) to have the doc branch and the source branch be different.  Is that what you meant?

Reply:
No, probably not.

I'm new to this and so I was guessing at some of the questions that bloom asked me. Then [CI failed](https://github.com/ros/rosdistro/pull/29515/checks?sha=53fe47bc5a9ec86fa29405f452f5ee0221632042) with this error:
```
AssertionError: ["In file '''/home/runner/work/rosdistro/rosdistro/noetic/distribution.yaml''': Could not
validate source entry for repo moveit_calibration with error [[[Could not validate repository with url
https://github.com/ros-planning/moveit_calibration.git and version 0.1.0 from entry at line 4013.
Error reason: Tags are not valid, but a tag 0.1.0 was found. Re:
https://github.com/ros/rosdistro/pull/20286]]]"] is not false
```
Based on the referenced PR, I switched the source version to a branch. I guess the recommended approach would be to make the doc version match?

Reply:
That's the most common pattern, yes.